helm-chart/templates/_helpers.tpl

```diff
@@ -0,0 +1,68 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "kubeshark.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
```

Comment:
`nameOverride` field does not exist in `values.yaml`.

Reply:
keeping it next to the [thread](https://github.com/kubeshark/kubeshark/pull/1406/files#r1288464716) conversation.